sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReshuffleTest.java

```diff
@@ -90,6 +95,29 @@ public void testJustReshuffle() {
 
   @Test
   @Category(RunnableOnService.class)
+  public void testReshuffleAndAssignTimestamps() {
+    PCollection<KV<String, Integer>> input = pipeline
+        .apply(Create.of(ARBITRARY_KVS)
+            .withCoder(KvCoder.of(StringUtf8Coder.of(), VarIntCoder.of())));
+
+    PCollection<KV<String, Integer>> output = input
+        .apply(Reshuffle.<String, Integer>of());
+
+    output.apply(WithTimestamps.of(new SerializableFunction<KV<String, Integer>, Instant>() {
```

Comment:
This can't be right: the returned PCollection of this .apply is never used. Can you also add some comments to the test about what it is testing?